web/src/app/modules/shared/components/presentation/resource-viewer/resource-viewer.component.ts

```diff
@@ -173,4 +180,36 @@ export class ResourceViewerComponent
   getLabel(label: string, length: number): string {
     return label.length > length ? label.substring(0, length) + '...' : label;
   }
+
+  resizeCursors() {
+    return {
+      topLeft: 'nw-resize',
+      topRight: 'ne-resize',
+      bottomLeft: 'sw-resize',
+      bottomRight: 'se-resize',
+      leftOrRight: 'ew-resize',
+      topOrBottom: 'ns-resize',
+    };
+  }
+
+  resizeEnd() {
+    this.zoom = Object.assign({}, defaultZoom); // update without layout recalc
+  }
+
+  resizeStart() {
+    this.startPosition = this.elRef.nativeElement.firstChild.firstChild.offsetWidth;
```

Comment:
Accessing children this way is a bit hard to read. A couple of options to make it readable would be to:
1) Assign these references to variables based on the element identifiers (`viewContainer` in this case).
2) Define ViewChild 
```
 @ViewChild('viewContainer')
  viewContainer: ElementRef;

  @ViewChild('resourceViewer')
  resourceViewer: ElementRef;

  @ViewChild('statusContainer')
  statusContainer: ElementRef;
```
and then use renderer (Renderer2) to update the style (`this.renderer.setStyle(this.viewContainer.nativeElement, 'width', `${leftSize}%`)`);

Reply:
Good point, Vikram - should have thought of that!